tools/codegen/base/release-plz.json

```diff
@@ -0,0 +1,15 @@
+{
+  "repository": "https://github.com/MarcoIeni/release-plz",
+  "website": "https://release-plz.ieni.dev/",
+  "tag_prefix": "release-plz-v",
+  "rust_crate": "${package}",
+  "asset_name": "${package}-${rust_target}.tar.gz",
+  "platform": {
+    "aarch64_macos": {},
+    "aarch64_windows": {},
+    "aarch64_linux_musl": {},
+    "x86_64_macos": {},
+    "x86_64_windows": {},
+    "x86_64_linux_gnu": {}
```

Comment:
Using x86_64_linux_musl here (and regenerate manifest) should fix the CI failure.

Reply:
Will do. Btw I had to remove `aarch64_linux_gnu` because generating the manifest failed with this error (forgot to mention it):
https://github.com/taiki-e/install-action/blob/cb5a3465812fbc1b425f9875b7294d08d6c4cb9f/tools/codegen/src/main.rs#L529-L535

I suppose there is no need in `gnu` variant when musl is available